driver-scala/src/test/scala/org/mongodb/scala/ApiAliasAndCompanionSpec.scala

```diff
@@ -65,7 +65,18 @@ class ApiAliasAndCompanionSpec extends BaseSpec {
       "SingleResultCallback",
       "SubjectProvider",
       "TransactionExample",
-      "UnixServerAddress"
+      "UnixServerAddress",
```

Comment:
Was it necessary to add all the primers?  I didn't think it would be.

Reply:
I added `testImplementation project(':driver-reactive-streams').sourceSets.test.output` to be able to call and reuse `SyncMongoClient.enableSleepAfterSessionClose`/`disableSleep` and `MainTransactionsTest.SESSION_CLOSE_TIMING_SENSITIVE_TESTS` from the Java code. As a result, `ApiAliasAndCompanionSpec` started complaining about all those new classes it started to see. I added only those it was complaining about (and instead of complaining about `PublisherHelpers` it was complaining about non-existent `ObservableHelpers` due to a deceiving line [`getSimpleName.replace("Publisher", "Observable")`](https://github.com/mongodb/mongo-java-driver/pull/722/files?file-filters%5B%5D=.scala#diff-d7d8702101580e51566b39b45c43b6e14fa4aa87263b27468521ae2d0499d025R146)).